etc/generate-openapi.sh

```diff
@@ -28,18 +28,22 @@ docker run --rm -it -u "$(id -u):$(id -g)" \
   --outfile /api/cli-extras.gen.yml \
   --type yaml
 
+# Strip certain tags to prevent duplicated and conflicting codegen.
+python3 ./etc/stripGroupTags.py ./api/cli.gen.yml > ./api/cli-stripped.gen.yml
```

Comment:
For portability, make this a docker command so `python3` isn't required to be in someone's path

Reply:
fixed